static/src/javascripts/projects/common/modules/identity/upsell/account-creation/AccountCreationFlow.js

```diff
@@ -0,0 +1,209 @@
+// @flow
+import React, { Component } from 'preact-compat';
+import arrowRight from 'svgs/icon/arrow-right.svg';
+import reqwest from 'reqwest';
+
+type AccountCreationFlowProps = {
+    csrfToken: string,
+    returnUrl: string,
+    accountToken: string,
+};
+
+type AccountCreationFormProps = {
+    csrfToken: string,
+    accountToken: string,
+    returnUrl: string,
+    onAccountCreated: () => {},
+};
+
+type AccountCreationFeaturesProps = {
+    returnUrl: string,
+};
+
+class AccountCreationFeatures extends Component<AccountCreationFeaturesProps> {
+    render() {
+        return (
+            <div>
+                <hr className="manage-account-small-divider" />
+                <h1 className="identity-title--small">
+                    Thanks for creating a Guardian account!
+                </h1>
+                <div className="identity-forms-message__body">
+                    <p>
+                        You are now signed in. Start exploring your benefits
+                        from our home page.
+                    </p>
+                </div>
+                <div className="identity-forms-message__body">
+                    <a
+                        className="manage-account__button manage-account__button--icon manage-account__button--main"
+                        data-link-name="complete-consents : cta-bottom"
+                        href={this.props.returnUrl}>
+                        Go to The Guardian
+                        <span
+                            // eslint-disable-next-line react/no-danger
+                            dangerouslySetInnerHTML={{
+                                __html: arrowRight.markup,
+                            }}
+                        />
+                    </a>
+                </div>
+            </div>
+        );
+    }
+}
+
+class AccountCreationForm extends Component<
+    AccountCreationFormProps,
+    {
+        password: string,
+        isLoading: boolean,
+        isError: boolean,
```

Comment:
As you're setting default values for the state of  `AccountCreationForm` I believe these should be maybe types.

```js
{
        password?: string,
        isLoading?: boolean,
        isError?: boolean,
}
```